ast/compile.go

```diff
@@ -3518,8 +3518,12 @@ func getGlobals(pkg *Package, rules []Var, imports []*Import) map[Var]*usedRef {
 	}
 
 	// Populate globals with imports.
-	for _, i := range imports {
-		globals[i.Name()] = &usedRef{ref: i.Path.Value.(Ref)}
+	for _, imp := range imports {
+		path := imp.Path.Value.(Ref)
+		if FutureRootDocument.Equal(path[0]) {
+			continue // ignore future imports
+		}
+		globals[imp.Name()] = &usedRef{ref: path}
 	}
```

Comment:
We used to treat future imports a bit too much like real imports: adding the name (e.g. `every` in `future.keywords.every`) to the `globals` variables, and make them OK vars to resolve. Now, when such a future kw is imported, any occurrences of `every` (staying in the example) would be keywords, not vars, so we don't need this at all. This is a small cleanup.